tests/integ/scala/test_function_suite.py

```diff
@@ -2829,25 +2829,100 @@ def test_array_append(session):
     reason="array_remove is not yet supported in local testing mode.",
 )
 def test_array_remove(session):
-    Utils.check_answer(
+    actual = session.createDataFrame([([1, 2, 4, 4, 3],), ([],)], ["data"])
+    actual = actual.select(array_remove(actual.data, 4))
+    expected = session.createDataFrame(
         [
-            Row("[\n  2,\n  3\n]"),
-            Row("[\n  6,\n  7\n]"),
+            Row("[\n  1,\n  2,\n  3\n]"),
+            Row("[]"),
         ],
-        TestData.array1(session).select(
-            array_remove(array_remove(col("arr1"), lit(1)), lit(8))
-        ),
-        sort=False,
```

Comment:
same comment about this test. Can you keep old test cases and add new ones below it please. This ensures that there is not behavior change.

Reply:
I rolled back the changes.